src/v2/Apps/Order/Utils/getStatusCopy.tsx

```diff
@@ -276,6 +276,9 @@ export const covidNote = (): React.ReactNode => {
   )
 }
 
+export const continueToInboxText =
```

Comment:
Q, why do we need it here and not directly in the component?

Reply:
We use it in two different places (congrats modal and Eigen's status page), always with other texts/components from this same file. I'd say the reasoning behind it was for centralizing it. Thoughts?

Reply:
Nah, it's okay, just wondering if the tradeoff was paid, but I can live with it